### DISCUSSION TITLE
Weird behaviour trying to create a table

### DISCUSSION THREAD

**Discussion by orchard800** [AntlerVC/firetable#1333] [Support / Q&A] [answered]
I'm getting these popups when I create a new table (as a sub-collection like clubs/members). When I add rows those popups won't stop! Meaning the page is unusable. Even if I navigate away and head back to it.
![Screenshot_20230703_224900_Chrome](https://github.com/rowyio/rowy/assets/84687635/37b151a1-bb7a-4e99-bc4b-d1e33deb5c12)
I've tried deleting the whole project in both rowy and firebase but keep getting the same result. I've even deleted my workspace and started again. Not a great first experience with rowy.

**Comment by BeeBombshell** [comment]
Hello, @orchard800! It seems that there is a bug when adding a subcollection where using a forward slash ("/") in the required path. We're currently investigating this on our end. Thanks for bringing this up!

**Comment by orchard800** [comment]
Hi, but without a slash it wouldn't be a sub-collection would it? It'd just be a root level collection called "clubs members". I'm trying to create a sub-collection of clubs called members.

**Reply by orchard800** [reply]
Update: I found the solution for creating subcollections here https://trm.to/vujls :-)

**Comment by BeeBombshell** [comment, marked as answer]
@orchard800, another way to create a table for a subcollection, you can indicate the path to the subcollection when setting up the table. See screenshots for reference.
![image](https://github.com/rowyio/rowy/assets/53828745/cb2b6705-d77c-4dd0-9212-c9145521cbe5)
![image](https://github.com/rowyio/rowy/assets/53828745/12ef9d21-fcef-462c-9835-e8c17eab8ca3)
